bot.py

```diff
@@ -637,134 +626,30 @@ def process_result(bot, update):
         reset_waiting_time(bot, player)
         do_play_card(bot, player, result_id)
 
-    if game in gm.chatid_games.get(chat.id, list()):
+    if game_is_running(game):
         send_async(bot, chat.id,
                    text=__("Next player: {name}", multi=game.translate)
                    .format(name=display_name(game.current_player.user)))
+        start_player_countdown(bot, game, job_queue)
 
 
 def reset_waiting_time(bot, player):
     """Resets waiting time for a player and sends a notice to the group"""
     chat = player.game.chat
 
-    if player.waiting_time < 90:
-        player.waiting_time = 90
+    if player.waiting_time < WAITING_TIME:
+        player.waiting_time = WAITING_TIME
         send_async(bot, chat.id,
-                   text=__("Waiting time for {name} has been reset to 90 "
+                   text=__("Waiting time for {name} has been reset to {time} "
```

Comment:
You're changing the translation string here... did you account for that in `unobot.pot` and locales? Just to make sure it doesn't break completely

Reply:
I don't know how unobot.pot works, but I have been playing in spanish just fine. The translation on the changed messages would have to be changed, yeah.